src/data/pages/projects-db.json

```diff
@@ -9,7 +9,7 @@
   },
   {
     "id": 1,
-    "project_name": "Mood board",
+    "project_name": "Mood board: lorem ipsum dolor sit amet, consectetur adipisicing elit.",
```

Comment:
Can we remove that long text? This app should looks the same as normal app, so no test text etc.

Reply:
Sure, will do, I left it just for convenience so that you can test the correspondent fix right away